pkg/kubernetes/apply.go

```diff
@@ -24,8 +24,12 @@ const AnnotationLastApplied = "kubectl.kubernetes.io/last-applied-configuration"
 // Orphaned returns previously created resources that are missing from the
 // local state. It uses UIDs to safely identify objects.
 func (k *Kubernetes) Orphaned(state manifest.List) (manifest.List, error) {
-	if !k.Env.Spec.InjectLabels {
-		return nil, fmt.Errorf(`spec.injectLabels is set to false in your spec.json. Tanka needs to add
+	if k.Env.Spec.InjectLabels {
+		return nil, fmt.Errorf(`spec.injectLabels for pruning is deprecated, please use spec.pruneMark instead.  See
```

Comment:
Just realized failing on this setting while keeping support for it in general is probably not going to work well.